interfaces/inc/c_pal/gballoc_ll.h

```diff
@@ -32,6 +32,8 @@ extern "C"
 
     MOCKABLE_FUNCTION(, void, gballoc_ll_print_stats);
 
+    MOCKABLE_FUNCTION(, int, gballoc_ll_set_decay, int64_t, decay_milliseconds);
```

Comment:
>decay [](http://example.com/codeflow?start=44&length=5)

decay is mushing together dirty and muzzy decay.
I would like it more if we'd keep both options and let the jemalloc implementation handle each one of them.

See my other comment about a "set_option".

Reply:
Done